dask/dataframe/csv.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import print_function, division, absolute_import
+
+from io import BytesIO
+from warnings import warn
+
+import numpy as np
+import pandas as pd
+
+from ..delayed import delayed
+from .io import from_delayed
+
+from ..bytes import read_bytes
+from ..bytes.compression import seekable_files, files as cfiles
+from ..utils import ensure_bytes
+
+
+delayed = delayed(pure=True)
+
+
+def bytes_read_csv(b, header, kwargs, dtypes=None, columns=None):
```

Comment:
Yes, one could arrange a dask.dataframe from many feather files, we wouldn't split those files though with a function like `read_bytes`.

In Parquet we wouldn't want all of the bytes, just the bytes for the particular column/row sets that we care about.  

CSV and Avro are both homogenous in that any sequential delimited set of bytes has more or less the same kind of information.  This is not true with Parquet, for which we would need to be more clever.